app/main.py

```diff
@@ -14,7 +14,6 @@ def run():
     logging.basicConfig(format=app.settings.LOGGING_FORMAT,
                         datefmt="%Y-%m-%dT%H:%M:%S",
                         level=app.settings.LOGGING_LEVEL)
-    logging.getLogger("pika").setLevel(logging.INFO)
```

Comment:
What happens if we don't set it at all? Does it default to the most verbose level or does it get switched off? I thought it wouldn't log anything unless we explicitly set the logger?

Reply:
I think this should be configurable. Can you create another setting in app.settings?

Reply:
:+1: for configuring this. We may want it at different levels dependent on the deployed environment.